tcg-useeffect/src/App.js

```diff
@@ -0,0 +1,16 @@
+import './App.css';
+import { useState, useEffect } from 'react';
+
+function App() {
```

Comment:
This is perfect on this file!

The rest of this should be in the initial commit, so that it's only this file that shows differently, but well done on getting the PR open.